data/mods/Magiclysm/Spells/attunements/Wither_Mage.json

```
@@ -52,5 +52,46 @@
     "base_energy_cost": 1200,
     "difficulty": 8,
     "extra_effects": [ { "id": "assassin_toxic_smoke", "hit_self": true }, { "id": "assassin_teleport" } ]
+  },
+  {
+    "type": "SPELL",
+    "id": "life_drain",
+    "name": "life_drain",
+    "description": "BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_",
```

```suggestion
    "name": "Life Drain",
    "description": "This spell takes some of an enemy's life, and in return gives you a temporary boost.",
```
It doesn't give HP, so "temporary".